task/s2i-angular-nginx/0.1/s2i-angular-nginx.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/displayName: s2i angular nginx
+    tekton.dev/categories: Image Build
+    tekton.dev/tags: image, build, angular, nginx
+    tekton.dev/pipelines.minVersion: '0.19'
+  name: s2i-angular-nginx
+  labels:
+    app.kubernetes.io/version: '0.1'
+spec:
+  params:
+    - default: '16'
+      description: The tag of nodejs version
+      name: NODEJS_VERSION
+      type: string
+    - default: .
+      description: The location of the path to run s2i from.
+      name: CONTEXT_PATH
+      type: string
+    - default: 'true'
+      description: >-
+        Verify the TLS on the registry endpoint (for push/pull to a non-TLS
+        registry)
+      name: TLSVERIFY
+      type: string
+    - description: Location of the repo where image has to be pushed
+      name: IMAGE
+      type: string
+    - default: >-
+        quay.io/buildah/stable:latest
+      description: The location of the buildah builder image
+      name: BUILDER_IMAGE
+      type: string
+    - name: INSTALL_COMMAND
+      description: command used to install dependencies
+      default: 'npm ci'
+      type: string
+    - name: BUILD_COMMAND
+      description: build command used to compile the source of the application
+      default: 'RUN npm run build -- --configuration=production'
+      type: string
+    - name: STORAGE_DRIVER
+      description: storage driver used to save the image to registry
+      default: vfs
+      type: string
+  results:
+    - description: Digest of the image just built.
+      name: IMAGE_DIGEST
+  steps:
+    - image: registry.access.redhat.com/ubi8@sha256:f20fb774c96377b793475021aca89909da74e9da05136eb6e824aa16f85f22db
+      name: generate
+      resources: {}
+      script: |
+        echo """
+        FROM node:$(params.NODEJS_VERSION) as node
+        WORKDIR /app
+        COPY $(params.CONTEXT_PATH) .
+        RUN $(params.INSTALL_COMMAND)
+        RUN $(params.BUILD_COMMAND)
+        FROM nginx:alpine
```

Comment:
Would be nice to parameterize the nginx image

Reply:
Do you mean we can't rely that there is a `/usr/share/nginx/html` directory in an alternative nginx image?